web_src/css/base.css

```diff
@@ -2,7 +2,8 @@
   /* fonts */
   --fonts-proportional: -apple-system, "Segoe UI", system-ui, Roboto, "Helvetica Neue", Arial;
   --fonts-monospace: ui-monospace, SFMono-Regular, "SF Mono", Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace, var(--fonts-emoji);
-  --fonts-emoji: "Apple Color Emoji", "Segoe UI Emoji", "Noto Color Emoji", "Twemoji Mozilla";
+  /* "Twemoji Mozilla" emoji font is widely used by browsers like Firefox, Pale Moon, and it is more likely up-to-dated than the system emoji font */
+  --fonts-emoji: "Twemoji Mozilla", "Apple Color Emoji", "Segoe UI Emoji", "Noto Color Emoji";
```

Comment:
See https://github.com/go-gitea/gitea/issues/34246#issuecomment-2824814747. Maybe we should set `--fonts-emoji` to a empty value, letting the browser choose the right font?

Reply:
Or better yet, just remove `--fonts-emoji` altogether. Users can still alter the other vars if they want to override emoji fonts.

Reply:
Maybe like this: https://github.com/go-gitea/gitea/pull/34263/commits/0b04bfb8c7715a140a2da7d2ff6f8f8d55c628b5. then the var is still kept and won't cause too many changes and the var could still be customized.

Reply:
I suppose a dummy won't hurt but I see no real reason to retain the variable either.

Reply:
There are 2 reasons in my mind:

1. There are not only one references to `--fonts-emoji`, I don't want to introduce too many changes or do more tests.
2. Keeping the dummy name makes it easier to revert in the future (if anything wrong). We can fully remove the dummy name in future releases if it has been widely tested and there wouldn't be any problem.